<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TriggerControlSetupWidgetClass</class>
 <widget class="QWidget" name="TriggerControlSetupWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>713</width>
    <height>584</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DummySetupWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <spacer name="m_qVerticalSpacer_Headline">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="m_qLabel_Headline">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Trigger Control Toolbox Configuration</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QGridLayout" name="m_qGridLayout_main">
     <item row="3" column="3">
      <widget class="QGroupBox" name="m_qGroupBox_Properties">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="title">
        <string>Properties</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
       <layout class="QGridLayout" name="m_qGridLayout_Properties">
        <item row="0" column="0">
         <widget class="QPushButton" name="m_qPushButton_Settings">
          <property name="text">
           <string>Settings</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="m_qPushButton_Connect">
          <property name="text">
           <string>Connect</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="m_qPushButton_Disconnect">
          <property name="text">
           <string>Disconnect</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QPushButton" name="m_qPushButton_About">
          <property name="text">
           <string>About</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QGroupBox" name="m_qGroupBox_Information">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Input</string>
       </property>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>561</width>
          <height>91</height>
         </rect>
        </property>
        <property name="title">
         <string>Input Information</string>
        </property>
        <widget class="QLabel" name="m_qLabel_Statusbar">
         <property name="geometry">
          <rect>
           <x>360</x>
           <y>20</y>
           <width>71</width>
           <height>16</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QWidget" name="">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>20</y>
           <width>240</width>
           <height>51</height>
          </rect>
         </property>
         <layout class="QFormLayout" name="formLayout_3">
          <item row="0" column="0">
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>118</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="m_qComboBox_AnalogSelect"/>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="m_qPushButton_RetrieveDigitalInfo">
            <property name="text">
             <string>Retrieve Digital Input</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="m_qPushButton_RetrieveAnalogInfo">
            <property name="text">
             <string>Retrieve Analog Input</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
       <zorder>groupBox_3</zorder>
       <zorder>groupBox</zorder>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QGroupBox" name="groupBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Output</string>
       </property>
       <widget class="QGroupBox" name="m_qGroupBox_Channels">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>191</width>
          <height>200</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>200</height>
         </size>
        </property>
        <property name="title">
         <string>Digital Output</string>
        </property>
        <widget class="QWidget" name="">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>30</y>
           <width>152</width>
           <height>132</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QRadioButton" name="m_qRadioButton_1">
                <property name="text">
                 <string/>
                </property>
                <property name="autoExclusive">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="m_qRadioButton_2">
                <property name="text">
                 <string/>
                </property>
                <property name="autoExclusive">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="m_qRadioButton_3">
                <property name="text">
                 <string/>
                </property>
                <property name="autoExclusive">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="m_qRadioButton_4">
                <property name="text">
                 <string/>
                </property>
                <property name="autoExclusive">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="m_qRadioButton_5">
                <property name="text">
                 <string/>
                </property>
                <property name="autoExclusive">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="m_qRadioButton_6">
                <property name="text">
                 <string/>
                </property>
                <property name="autoExclusive">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="m_qRadioButton_7">
                <property name="text">
                 <string/>
                </property>
                <property name="autoExclusive">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="m_qRadioButton_8">
                <property name="text">
                 <string notr="true"/>
                </property>
                <property name="autoExclusive">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>1</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>2</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>3</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>4</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>5</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_5">
                <property name="text">
                 <string>6</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string>7</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_8">
                <property name="text">
                 <string>8</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>13</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QRadioButton" name="m_qRadioButton_9">
                <property name="text">
                 <string/>
                </property>
                <property name="autoExclusive">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="m_qRadioButton_10">
                <property name="text">
                 <string/>
                </property>
                <property name="autoExclusive">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="m_qRadioButton_11">
                <property name="text">
                 <string/>
                </property>
                <property name="autoExclusive">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="m_qRadioButton_12">
                <property name="text">
                 <string/>
                </property>
                <property name="autoExclusive">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="m_qRadioButton_13">
                <property name="text">
                 <string/>
                </property>
                <property name="autoExclusive">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="m_qRadioButton_14">
                <property name="text">
                 <string/>
                </property>
                <property name="autoExclusive">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="m_qRadioButton_15">
                <property name="text">
                 <string/>
                </property>
                <property name="autoExclusive">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="m_qRadioButton_16">
                <property name="text">
                 <string/>
                </property>
                <property name="autoExclusive">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QLabel" name="label_9">
                <property name="text">
                 <string>9</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_14">
                <property name="text">
                 <string>10</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_12">
                <property name="text">
                 <string>11</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_13">
                <property name="text">
                 <string>12</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_11">
                <property name="text">
                 <string>13</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_16">
                <property name="text">
                 <string>14</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_15">
                <property name="text">
                 <string>15</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_10">
                <property name="text">
                 <string>16</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>18</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="m_qPushButton_Send">
              <property name="text">
               <string>Send</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>28</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <zorder>layoutWidget</zorder>
        <zorder>label</zorder>
        <zorder>label_2</zorder>
        <zorder>label_3</zorder>
        <zorder>label_4</zorder>
        <zorder>label_5</zorder>
        <zorder>label_6</zorder>
        <zorder>label_7</zorder>
        <zorder>label_8</zorder>
        <zorder>label_9</zorder>
        <zorder>label_10</zorder>
        <zorder>label_11</zorder>
        <zorder>label_12</zorder>
        <zorder>label_13</zorder>
        <zorder>label_14</zorder>
        <zorder>label_15</zorder>
        <zorder>label_16</zorder>
        <zorder>verticalSpacer</zorder>
        <zorder>horizontalSpacer</zorder>
        <zorder>horizontalSpacer_2</zorder>
        <zorder>groupBox_4</zorder>
       </widget>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>20</y>
          <width>291</width>
          <height>141</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>Analog Output</string>
        </property>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>20</y>
           <width>248</width>
           <height>104</height>
          </rect>
         </property>
         <layout class="QFormLayout" name="formLayout_2">
          <item row="0" column="0">
           <layout class="QFormLayout" name="formLayout">
            <item row="0" column="0">
             <widget class="QRadioButton" name="m_qRadioButton_motor1">
              <property name="text">
               <string>Channel 1</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QRadioButton" name="m_qRadioButton_motor2">
              <property name="text">
               <string>Channel 2</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QRadioButton" name="m_qRadioButton_motor3">
              <property name="text">
               <string>Channel 3</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QRadioButton" name="m_qRadioButton_motor4">
              <property name="text">
               <string>Channel 4</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="1">
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="1" column="0">
             <widget class="QLCDNumber" name="m_qAnalogDisp"/>
            </item>
            <item row="0" column="1" rowspan="2">
             <widget class="QDial" name="dial">
              <property name="maximum">
               <number>65535</number>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QPushButton" name="m_qPushButton_Sendanalog">
              <property name="text">
               <string>Send</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>170</y>
          <width>291</width>
          <height>51</height>
         </rect>
        </property>
        <property name="title">
         <string>Connect Channel</string>
        </property>
        <widget class="QWidget" name="">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>20</y>
           <width>175</width>
           <height>25</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QComboBox" name="m_qComboBox_ChannelList"/>
          </item>
          <item>
           <widget class="QPushButton" name="m_qPushButton_ConnectChannel">
            <property name="text">
             <string>Connect to Output</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
       <zorder>m_qGroupBox_Channels</zorder>
       <zorder>groupBox_4</zorder>
       <zorder>m_qGroupBox_Information</zorder>
       <zorder>groupBox_2</zorder>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>dial</sender>
   <signal>valueChanged(int)</signal>
   <receiver>m_qAnalogDisp</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>291</x>
     <y>416</y>
    </hint>
    <hint type="destinationlabel">
     <x>294</x>
     <y>370</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
